src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/EditorInProcess.cs

```diff
@@ -132,6 +142,242 @@ public async Task SetUseSuggestionModeAsync(bool value, CancellationToken cancel
             }
         }
 
+        #region Navigation bars
+
+        public async Task ExpandProjectNavigationBarAsync(CancellationToken cancellationToken)
+        {
+            await ExpandNavigationBarAsync(index: 0, cancellationToken);
+        }
+
+        public async Task ExpandTypeNavigationBarAsync(CancellationToken cancellationToken)
+        {
+            await ExpandNavigationBarAsync(index: 1, cancellationToken);
+        }
+
+        public async Task ExpandMemberNavigationBarAsync(CancellationToken cancellationToken)
+        {
+            await ExpandNavigationBarAsync(index: 2, cancellationToken);
+        }
+
+        public async Task ExpandNavigationBarAsync(int index, CancellationToken cancellationToken)
```

Comment:
make private?

Reply:
➡️ Eliminated the intermediate callers and left this public